app/OpenGraphImage.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App;
+
+use GDText\Box;
+use GDText\Color;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+
+class OpenGraphImage
+{
+    const PADDING = 50;
+    const GUTTER = 15;
+
+    protected $fileName;
+
+    /**
+     * Generate an open graph image file name.
+     *
+     * @param string $uniqueId A unique key to identify the image by.
+     * @param string $name Name of the file.
+     */
+    public static function makeFileName($uniqueId, $name): string
+    {
+        return "{$uniqueId}_" . Str::slug($name, '-') . '.png';
+    }
```

Comment:
Tlint complains about this since the method is static? Should I swap and ignore Tlint?

Reply:
But I guess this doesn't really matter if I create the `generate()` method (that you suggested above) thats being called statically ... I could get rid of the constructor all together.

Reply:
So, TLint is wrong here, apparently :)

However, rather than use a static method here, let's instead keep the constructor, accept all 3 parameters, and add a non-static `generate` method:

```
(new OpenGraphImage($this->packageName, $this->packageAuthor, $this->packageOgImageName))->generate();
```

Every other method in this class, then, can be `protected`, since they're only used internally to the class.